docs/examples/vllm/TPU/lws.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: leaderworkerset.x-k8s.io/v1
```

Comment:
Could you highlight the major difference to deploy with GPU? Thanks.

Reply:
Added a section on the TPU read me highlighting the differences